api/src/plugins/fastify-sentry.ts

```diff
@@ -0,0 +1,23 @@
+import { init, captureException } from '@sentry/node';
+import { FastifyInstance, FastifyPluginOptions } from 'fastify';
+import fp from 'fastify-plugin';
+
+const fastifySentry = (
+  fastify: FastifyInstance,
+  options: FastifyPluginOptions,
+  done: (err?: Error) => void
+) => {
```

Comment:
```suggestion
const fastifySentry: FastifyPluginCallback = (fastify, options, done) => {
```
Same thing, but a little neater.

Reply:
the plugin has wrong type for options.

```ts
Options extends FastifyPluginOptions = Record<never, never>,
```

Reply:
```suggestion
const fastifySentry: FastifyPluginCallback<{ dsn: string }> = (
  fastify,
  options,
  done
) => {
```
Ah, so it is, thanks. This version should work and add a little type safety.